app/mixins/item_actions.js

```diff
@@ -186,6 +197,10 @@ export default Ember.Mixin.create(AsyncMixin, {
       }
     },
 
+    onProcessingDestinationChange(destination) {
```

Comment:
`setProcessingDestination` makes more sense as a name

`onXXX` would mean that it has already happened, but this method actually does the set